src/main/webapp/app/buildagent/shared/number-input/number-input.component.spec.ts

```
@@ -0,0 +1,218 @@
+    beforeEach(() => {
+        fixture = TestBed.createComponent(NumberInputComponent);
+        component = fixture.componentInstance;
+
+        // Set up required inputs before detectChanges
+        fixture.componentRef.setInput('value', 5);
+        fixture.componentRef.setInput('minValue', 1);
+        fixture.componentRef.setInput('maxValue', 10);
+        fixture.componentRef.setInput('disabled', false);
+
+        fixture.detectChanges();
+    });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)